ext/RMagick/rmimage.c

```diff
@@ -10279,11 +11307,17 @@ Image_quantize(int argc, VALUE *argv, VALUE self)
             if (rb_obj_is_kind_of(argv[2], Class_DitherMethod))
             {
                 VALUE_TO_ENUM(argv[2], quantize_info.dither_method, DitherMethod);
+#if !defined(IMAGEMAGICK_7)
```

Comment:
It would be weird to do that and for the the `!` is pretty normal but I can add an `IMAGEMAGICK_6` define if you would prefer that. If you really want that could I do that in a new PR because it will also touch other files. You decide?

Reply:
Okay will do that in a new PR after this one is merged.